### ISSUE TITLE
Ability to show/hide icons

### ISSUE THREAD

**Issue by RuizuKun-Dev** [1ForeverHD/TopbarPlus#126]
I made sure to read API documentation first before coming here, I am looking for a simple functionality of Showing and Hiding an Icon for example, I might not want a button to be hidden in a dropdown at one point in time and then at a later point in time I'd want to show the button again

I understand I can create a new button instead however this is wasteful and a hassle to setup, and would be easier to just reuse the icon

In short I'd like the ability to show/hide icons like so :Hide(), :Show(), without disconnecting my events and destroying Child Icons etc
